libxrandr install error: configure error: C compiler cannot create executables

## Before you submit an issue
Please look up our issues, maybe your problem was mentioned before.

## Description
Provide a short description of your problem here.

If you executed a specific command:
```shell
crew install opera libxrandr
```

Output:
```shell
gcc (GCC) 8.3.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
Rename all *.la files to *.la_tmp
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking build system type... x86_64-cros-linux-gnu
checking host system type... x86_64-cros-linux-gnu
checking how to print strings... printf
checking for style of include used by make... GNU
checking for x86_64-cros-linux-gnu-gcc... x86_64-cros-linux-gnu-gcc
checking whether the C compiler works... no
configure: error: in `/usr/local/tmp/crew/libXrandr-1.5.2.tar.gz.dir/libXrandr-1.5.2':
configure: error: C compiler cannot create executables
See `config.log' for more details
libxrandr failed to install: `./configure --prefix=/usr/local --libdir=/usr/local/lib64 --mandir=/usr/local/share/man --build=x86_64-cros-linux-gnu --host=x86_64-cros-linux-gnu --target=x86_64-cros-linux-gnu` exited with 77
```

--- a/packages/libxrandr.rb
+++ b/packages/libxrandr.rb
@@ -11,6 +11,7 @@ class Libxrandr < Package
 
   depends_on 'libxrender'
   depends_on 'libx11'
+  depends_on 'libxext'
   depends_on 'llvm' => ':build'
 
   def self.build

